modules/tgw/README.md

```diff
@@ -340,3 +340,29 @@ tgw/spoke:
           stage: prod
           environment: usw2
 ```
+
+## Destruction
+
+When destroying Transit Gateway components, order of operations matters. Always destroy any removed `tgw/spoke` components before removing a connection from the `tgw/hub` component.
+
+The `tgw/hub` component creates map of VPC resources that each `tgw/spoke` component references. If the required reference is removed before the `tgw/spoke` is destroyed, Terraform will fail to destroy the given `tgw/spoke` component.
```

Comment:
Maybe consider suggesting using workflows to document how to create/destroy in the TGW network.

Reply:
added!